src/SDKs/SqlManagement/Sql.Tests/SqlDatabaseAgentScenarioTests.cs

```diff
@@ -0,0 +1,601 @@
+                    {
+                        Location = server.Location,
+                        DatabaseId = db1.Id
+                    });
+
+                    // Update agent tags
+                    agent1 = sqlClient.JobAgents.CreateOrUpdate(resourceGroup.Name, server.Name, agentName, new JobAgent
+                    {
+                        Location = server.Location,
+                        DatabaseId = db1.Id,
+                        Tags = new Dictionary<string, string>()
+                    {
+                        { "tagKey1", "TagValue1" }
+                    }
+                    });
+
+                    // Get agent
+                    agent1 = sqlClient.JobAgents.Get(resourceGroup.Name, server.Name, agentName);
+
+                    // List agents
+                    var agents = sqlClient.JobAgents.ListByServer(resourceGroup.Name, server.Name);
+
+                    // Delete agent
+                    sqlClient.JobAgents.Delete(resourceGroup.Name, server.Name, agentName);
+                }
+                finally
+                {
+                    // Clean up
+                    context.DeleteResourceGroup(resourceGroup.Name);
+                }
+            }
+        }
+
+        /// <summary>
+        /// Tests end to end job credential
+        /// </summary>
+        [Fact]
+        public void TestCreateUpdateDropJobCredential()
+        {
+            using (SqlManagementTestContext context = new SqlManagementTestContext(this))
+            {
+                ResourceGroup resourceGroup = context.CreateResourceGroup();
+                Server server = context.CreateServer(resourceGroup);
+                SqlManagementClient sqlClient = context.GetClient<SqlManagementClient>();
+
+                try
+                {
+                    // Create database only required parameters
+                    string dbName = SqlManagementTestUtilities.GenerateName();
+                    var db1 = sqlClient.Databases.CreateOrUpdate(resourceGroup.Name, server.Name, dbName, new Database()
+                    {
+                        Location = server.Location,
+                    });
+                    Assert.NotNull(db1);
+
+                    // Create agent
+                    string agentName = "agent";
+
+                    JobAgent agent = sqlClient.JobAgents.CreateOrUpdate(resourceGroup.Name, server.Name, agentName, new JobAgent
+                    {
+                        Location = server.Location,
+                        DatabaseId = db1.Id
+                    });
+
+                    // Create credential
+                    JobCredential credential = sqlClient.JobCredentials.CreateOrUpdate(resourceGroup.Name, server.Name, agent.Name, SqlManagementTestUtilities.DefaultLogin, new JobCredential
+                    {
+                        Username = "cloudsa",
+                        Password = "Yukon900!"
```

Comment:
You could use
```
                Username = SqlManagementTestUtilities.DefaultLogin,
                Password = SqlManagementTestUtilities.DefaultPassword,
```

Reply:
Never mind, you update below. Can you make this initial credential more fake looking, like just "a" and "b"

Reply:
Sure